src/components/form-draft/publish.vue

```diff
@@ -188,6 +254,14 @@ export default {
     },
     "problem": {
       "409_6": "The version name of this Draft conflicts with a past version of this Form or a deleted Form. Please use the field below to change it to something new or upload a new Form definition."
+    },
+    "dataset": {
+      "introduction": {
+        "full": "{inAddition} publishing this Form definition will make the following changes to this Project:",
+        "inAddition": "In addition,"
+      },
+      "newDataset": "A new Dataset {datasetName} will be created.",
+      "newProperty": "In Dataset {datasetName}, a new field {propertyName} will be created."
```

Comment:
field -> property

I think we discussed this a couple weeks ago -- I think it's helpful to be consistent.